src/test/memory-leak/runMemlabTests.js

```diff
@@ -0,0 +1,15 @@
+const { run } = require('@memlab/api');
+
+const tests = ['./tests/toggleMobileMenu', './tests/sliderScroll'];
```

Comment:
hardcode, you dont know what tests will be in the folder, https://nodejs.org/api/fs.html, you can use this for check all test files, but firstly you need to find another approach to execute memory leak tests, maybe library already have this option